tools/mason/MasonSearch.chpl

```diff
@@ -85,15 +88,22 @@ proc isHidden(name : string) : bool {
 proc findLatest(packageDir) {
   var ret = new VersionInfo(0, 0, 0);
   const suffix = ".toml";
-
+  var foundValidVersion = false;
```

Comment:
This variable is no longer needed.

Reply:
I am sorry for completely missing that. I updated the code.